src/test/java/com/google/sps/util/GmapsHelperTest.java

```diff
@@ -87,6 +89,20 @@ public void testSearchNearbyCategoryGroupReturnsValidJSON()
     Assert.assertEquals(true, isJSONValid(results));
   }
 
+  @Test
+  public void testrouteDistanceBetweenPoints()
+      throws ApiException, InterruptedException, IOException {
+
+    // Google TC3 to Denny's (1 mile distance)
+    DistanceMatrixRow[] results =
+        gmaps.routeDistanceBetweenPoints(
+            new LatLng(37.402971, -122.032619), new LatLng(37.396145, -122.027790));
+    System.out.println("Distance: " + results[0].elements[0].distance);
+    System.out.println("Duration: " + results[0].elements[0].duration);
+    System.out.println("durationInTraffic: " + results[0].elements[0].durationInTraffic);
+    System.out.println("durationInTraffic: " + results[0].elements[0].status);
```

Comment:
can you add assertion to verify data from `results`?

Reply:
Ahh yeah sorry this is actually just test code that I thought I had removed but I guess not.